unredirect@vaina.lt/extension.js

```diff
@@ -1,29 +1,26 @@
-import Meta from 'gi://Meta';
-
 export default class UnredirectExtension {
     constructor() {
-        this.orig_enable_unredirect_for_display = null;
+        this.orig_enable_unredirect = null;
     }
 
     enable() {
-        if (this.orig_enable_unredirect_for_display == null) {
-            console.debug("Monkey patching Meta.enable_unredirect_for_display(display) function");
-            this.orig_enable_unredirect_for_display = Meta.enable_unredirect_for_display;
-            Meta.enable_unredirect_for_display = function(display) {
-                // should I check for display?
-                console.debug("Ignoring invocation of Meta.enable_unredirect_for_display(display)");
+        if (this.orig_enable_unredirect == null) {
+            console.debug("Monkey patching global.compositor.enable_unredirect() function");
+            this.orig_enable_unredirect = global.compositor.enable_unredirect;
+            global.compositor.enable_unredirect_for_display = function () {
```

Comment:
Seems to me this should be `enable_unredirect` instead of `enable_unredirect_for_display` as well?

Reply:
I agree.

Reply:
Overall direction is ok https://gjs.guide/extensions/upgrading/gnome-shell-48.html#meta

Reply:
```suggestion
            global.compositor. enable_unredirect = function () {
```
I strongly suggest this change because it may appear to be working by accident and retest it again. Thank you for the effort. I'm spending very little time on this extension. Contributions keep it afloat.